package.json

```diff
@@ -99,7 +99,7 @@
     "webpack-hot-middleware": "^2.13.2"
   },
   "peerDependencies": {
-    "draft-js-plugins-editor": "2.0.0-beta10",
+    "draft-js-plugins-editor": "2.0.0-rc2",
```

Comment:
Actually, I still get peer dependency warnings when I run with `-beta10` instead of `-rc2`. They are just warnings though